fragalysis_api/xcimporter/align.py

```diff
@@ -146,7 +149,11 @@ def align_to_reference(self, in_file, reference, out_dir):
                 print(i)
                 s2 = time.time()
                 map = Xmap.from_file(file=Path(os.path.join(dir, f'{base}{ext}')))
+                array = np.array(map.xmap, copy=False)
```

Comment:
Specifically Here: we remove all nan values from map files such that the alignment from pandda gemmi can take place correctly. This was a annoying bug that is now fixed and greatly improves the accessibility of the api.